helm/libero-reviewer/requirements.lock

```diff
@@ -2,5 +2,8 @@ dependencies:
 - name: postgresql
   repository: https://kubernetes-charts.storage.googleapis.com/
   version: 6.2.1
-digest: sha256:b1791c8964a84e7e17d008a26bd2ab5f6e49e167107746d048bdab25ed078569
-generated: 2019-09-16T15:51:54.715208129+01:00
+- name: rabbitmq
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  version: 6.12.1
```

Comment:
`persistence.enabled` is `true` by default, do you want that? Would check it's consistent with the `postgresql` value